src/components/opoTable/opoTable.js

```diff
@@ -14,10 +16,20 @@ export default function OpoTable({
   opos,
   title,
 }) {
-  if (!opos.length) return null;
 
+
+const columns = useMemo(() => {
+  const cols = [
```

Comment:
Let's clean up the `map`/`filter` below and have the col logic here. Something like

```
const columns = [
    ...(inState ? ["name", "region"] : ["state", "name"]) ,
    "tier",
    "donors",
    "shadow",
    "investigation",
  ];
```
maybe?